extension/encoding/awslogsencodingextension/internal/unmarshaler/cloudtraillog/unmarshaler.go

```diff
@@ -27,14 +27,32 @@ var _ unmarshaler.AWSUnmarshaler = (*CloudTrailLogUnmarshaler)(nil)
 
 // UserIdentity represents the user identity information in CloudTrail logs
 type UserIdentity struct {
-	Type             string `json:"type"`
-	PrincipalID      string `json:"principalId"`
-	ARN              string `json:"arn"`
-	AccountID        string `json:"accountId"`
-	AccessKeyID      string `json:"accessKeyId"`
-	UserName         string `json:"userName"`
-	UserID           string `json:"userId"`
-	IdentityStoreARN string `json:"identityStoreArn"`
+	Type             string          `json:"type"`
+	PrincipalID      string          `json:"principalId"`
+	ARN              string          `json:"arn"`
+	AccountID        string          `json:"accountId"`
+	AccessKeyID      string          `json:"accessKeyId"`
+	UserName         string          `json:"userName"`
+	UserID           string          `json:"userId"`
+	IdentityStoreARN string          `json:"identityStoreArn"`
+	InvokedBy        string          `json:"invokedBy"`
+	SessionContext   *SessionContext `json:"sessionContext"`
+}
+
+// SessionContext if request was made with temporary security credentials,
+// provides information about the session created for credentials.
+type SessionContext struct {
+	Attributes    map[string]any `json:"attributes"`
```

Comment:
Can you remove this `any`? It decreases performance, and it will always stop us from following semantic conventions, if any. [Attributes are known](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-event-reference-user-identity.html):

```
attributes – The attributes for the session.

    creationDate – The date and time when the temporary security credentials were issued. Represented in ISO 8601 basic notation.

    mfaAuthenticated – The value is true if the root user or IAM user who used their credentials for the request also authenticated with an MFA device; otherwise, false.
```

Reply:
Good input, I completed this through commit https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/43584/commits/e2d91028211a38da86e5a1855b9c0c822b9482e5